documentation/docker.md

```diff
@@ -19,7 +19,7 @@ It's worth a quick reminder that Containers are created from Images. [Docker's o
 
 For both stages, we:
 
-- Use the [official Docker Ruby image](https://hub.docker.com/_/ruby), based off [Alpine Linux `3.11`, with Ruby `2.7.2` included](https://github.com/docker-library/ruby/blob/a564feaaee4c8647c299ab11d41498468bb9af7b/2.7/alpine3.11/Dockerfile)
+- Use the [official Docker Ruby image](https://hub.docker.com/_/ruby), based off [Alpine Linux `3.11`, with Ruby `2.7.2` included](https://github.com/docker-library/ruby/blob/master/2.7/alpine3.11/Dockerfile)
```

Comment:
Can we not detail the versions here? This may quickly get out of date

Reply:
Explained about how we choose the image tag, according to the version of Ruby.